cmd/root.go

```diff
@@ -485,7 +212,9 @@ var newlineReplaceRegex = regexp.MustCompile("\r?\n")
 // updateIssue compares each field of a GitHub issue to a JIRA issue; if any of them
 // differ, the differing fields of the JIRA issue are updated to match the GitHub
 // issue.
-func updateIssue(ghIssue github.Issue, jIssue jira.Issue, ghClient github.Client, jClient jira.Client) error {
+func updateIssue(config lib.Config, ghIssue github.Issue, jIssue jira.Issue, ghClient github.Client, jClient jira.Client) error {
```

Comment:
same: all of these functions that are *not* directly concerned with configuring and launching the command should be outside of this file and package. this function should be in lib.